src/dsp_tools/cli/entry_point.py

```diff
@@ -67,12 +67,12 @@ def run(args: list[str]) -> None:
         )
         success = call_requested_action(parsed_arguments)
     except BaseError as err:
-        logger.exception("The process was terminated because of an Error:")
+        logger.error("The process was terminated because of an Error:")
```

Comment:
why don't you log the exception information any more? In loguru, it's the same as in the standard library:
 - logger.error(): without stack trace
 - logger.exception(): with stack trace 

cf. https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.exception